README.md

```diff
@@ -2,6 +2,7 @@
 
 [![docs](https://docs.rs/lewton/badge.svg)](https://docs.rs/crate/lewton)
 [![crates.io](https://img.shields.io/crates/v/lewton.svg)](https://crates.io/crates/lewton)
+[![Actions Status](https://github.com/RustAudio/lewton/workflows/lewton/badge.svg)](https://github.com/RustAudio/lewton/actions)
```

Comment:
Can you not add this badge? I don't think such badges are useful.

Reply:
Done, thanks!